index.js

```diff
@@ -151,6 +151,8 @@ function init (ssb, config) {
     link: {
       create: scuttle.link.create
     },
-    findByFeedId: scuttle.link.findGroupByFeedId
+    findByFeedId: scuttle.link.findGroupByFeedId,
+    get: keystore.group.get,
```

Comment:
:fire: you've added these to `manifest`, so these methods will only be callable if your sbot is local, not over e.g. ssb-client connections. Is that deliberate?

Reply:
Nope, I'll add a test to ensure that we're exporting all of our methods.